### ISSUE TITLE
Streamline the "Applications" page

### ISSUE THREAD

**Issue by joepavitt** [FlowFuse/flowfuse#4737]
### Description

<img width="1440" height="1024" alt="Image" src="https://github.com/user-attachments/assets/6157a9c6-9752-443a-93b5-e6fffccb93c7" />

### Related Issues

Pairs with https://github.com/FlowFuse/flowfuse/issues/4729

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

**Comment by joepavitt** [comment]
### Current UI

<img width="1728" alt="Image" src="https://github.com/user-attachments/assets/5c0f8827-b635-44d7-9739-e42732beeead" />

Current page has far too many CTAs. Need to strip this back, address the main points of this page, and make navigation _to the Application_ far clearer than it is currently.

**Comment by joepavitt** [comment]
<img width="1728" alt="Image" src="https://github.com/user-attachments/assets/c1e0306c-63b2-41cc-bcbb-bf92d9ff85e4" />

The challenge here is, most users want to go to the Editor, on this page, that's enabled by exposing lots of CTAs, if we refine the actions to just Applications, then we lose the quick path to desired destination.

The second highest traffic though is to "Instances" views, which is what I'd expect, and will only increased further if we remove CTAs from this view. 

As such, I think our better option here is to introduce a new "Overview"/"Homepage", which shows recent activity in the team, most recently deployed/modified Instances, or maybe even a "Favourites" for quick access to the things that users will care about.

**Comment by joepavitt** [comment]
<img width="1443" alt="Image" src="https://github.com/user-attachments/assets/4d7df45f-4358-454a-9256-419424e269c5" />

This shows that he majority of users (nearly 50%) are actually using the shortcuts to the Instance/Device pages from the current Applications overview, this is higher than expected, s does reinforce the need for a way of easily navigating to resources from the landing screen.

**Comment by joepavitt** [comment]
Chatting with @gstout52 - the challenge we have now is that we are treating "Applications" as a home page, trying to surface quick access to Instances, Pipelines, etc. That in turn is making the view incredibly busy.

So my vote would be that "Applications" moves under "Instances" int he left-side navigation, then we have a new "Overview" page which helps users get the information they're likely to want and provide quick access to relevant Instances, etc.

**Comment by knolleary** [comment]
Just to add, the current home page implementation isn't scaling well from a performance POV - #5520

**Comment by gstout52** [comment]
Scheduling this for 2.19 to give us some time to prioritize observability, NR onboarding, and design the homepage.

**Comment by gstout52** [comment]
@joepavitt Can you provide a link to the PH insight you shared above? We can use that to provide a maximally useful homepage since we already know what users are looking for from the landing page.

**Comment by joepavitt** [comment]
@gstout52 https://eu.posthog.com/project/2209/dashboard/148295

**Comment by joepavitt** [comment]
<img width="1000" height="652" alt="Image" src="https://github.com/user-attachments/assets/789baa85-1902-4f07-9f7f-cc2b34ba88d8" />

[PostHog Insight](https://eu.posthog.com/project/2209/insights/SQOfIjzg)

Surprised by the number of people that use the mini-shortcuts on the right to quickly navigate to instances/devices, etc. for a given Application.

I had a plan to remove them, but will instead keep them and revamp the section below.

**Comment by joepavitt** [comment]
<img width="633" height="513" alt="Image" src="https://github.com/user-attachments/assets/88d00ba0-b6ef-4b8d-8203-43b7cbb514be" />

This is the number of visitors to that page which then click the "Open Editor" or "Open Dashboard" shortcut

**Comment by joepavitt** [comment]
<img width="646" height="512" alt="Image" src="https://github.com/user-attachments/assets/321a8d9e-eeff-44d8-bde0-d2a75ba75edc" />

PostHog Insight: https://eu.posthog.com/project/2209/insights/FDkwl1Wt

The most damning statistic of this page is that the lowest conversion is to actually open/view an Application. This will be a core metric to meaure once this revamp has been done.

**Comment by joepavitt** [comment]
@cstns I've updated the main description with the new design, inspiration taken from the new home page, whilst keeping the quick shortcuts given that users are actively using them.

**Comment by cstns** [comment]
I assume the search bar would only search through applications from now on. Users might feel a bit cheated that we left them with the only place (they knew of) they could search through apps/instances/devices in one go.

We should try and point users to the global search bar as a replacement, maybe even include it in the product tour somehow.

_(afterthought: include pipelines and device groups in the global search results)_

**Comment by joepavitt** [comment]
It's a good point, for now, I say yes, filter by applications only. Let's see how that performs. Global search offers the full search capability, and is, technically, on th same page.

**Comment by joepavitt** [comment]
@cstns assigning this to you as we can proceed with the dev work.

**Comment by cstns** [comment]
should I prioritize it before or after the tables UI ?

**Comment by joepavitt** [comment]
If there is progress to be made on the Tables work, then proceed with Tables. Wasn't sure on the status of the API work on that side

**Comment by gstout52** [comment]
Why is this blocked? Pending Tables work? @joepavitt

**Comment by joepavitt** [comment]
@cstns was meant to detail in here the reasons why

**Comment by joepavitt** [comment]
So, we could proceed with this work (and arguably should), but it is blocked from working _properly_ because of: https://github.com/FlowFuse/flowfuse/issues/5721, which means that the counters for the states (running, error, etc) are inaccurate. Given that issue itself is blocked, I see no reason why we shouldn't press forward with this now anyway, knowingly showing the wrong numbers, but it;'s still a better UX than we currently have.

**Comment by cstns** [comment]
Yes, my bad! was meaning to update the task after the engineering meeting but forgot after lunch. 

Prioritization wise, should I take this on after the product tours and before resuming the tables offering?

**Comment by joepavitt** [comment]
@cstns I'd say Product Tour, Tables, then this.
